File: binance4j-blvt/src/main/java/com/binance4j/blvt/dto/SubscriptionResponse.java
package com.binance4j.blvt.dto;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;

/**
 * BVLT subscription response.
 * 
 * @param id        Subscription id.
 * @param status    Subscription status. S("success"), P("pending"), and
 *                  F("failure").
 * @param tokenName Token name.
 * @param amount    Subscribed token amount.
 * @param cost      Subscription cost.
 * @param timestamp Timestamp in ms.
 */
@ApiModel("BVLT subscription response.")
public record SubscriptionResponse(@ApiModelProperty("Subscription id.") long id,
		@ApiModelProperty("Subscription status. S(\"success\"), P(\"pending\"), and F(\"failure\").") String status,
		@ApiModelProperty("Token name.") String tokenName, @ApiModelProperty("Subscribed token amount.") String amount,
		@ApiModelProperty("Subscription cost.") String cost, @ApiModelProperty("Timestamp in ms.") long timestamp) {
}

File: binance4j-blvt/src/main/kotlin/com/binance4j/blvt/dto/SubscriptionResponse.kt
package com.binance4j.blvt.dto

import io.swagger.annotations.ApiModel
import io.swagger.annotations.ApiModelProperty

/**
 * BLVT subscription response.
 *
 * @property id        Subscription id.
 * @property status    Subscription status. S("success"), P("pending"), and F("failure").
 * @property tokenName Token name.
 * @property amount    Subscribed token amount.
 * @property cost      Subscription cost.
 * @property timestamp Timestamp in ms.
 */
@ApiModel("BLVT subscription response.")
data class SubscriptionResponse(
    @ApiModelProperty("Subscription id.")
    var id: Long? = null,
    @ApiModelProperty("Subscription status. S(\"success\"), P(\"pending\"), and F(\"failure\").")
    var status: String? = null,
    @ApiModelProperty("Token name.")
    var tokenName: String? = null,
    @ApiModelProperty("Subscribed token amount.")
    var amount: String? = null,
    @ApiModelProperty("Subscription cost.")
    var cost: String? = null,
    @ApiModelProperty("Timestamp in ms.")
    var timestamp: Long? = null
)